src/freenet/clients/http/bookmark/Bookmark.java

```diff
@@ -18,7 +18,7 @@ public String getVisibleName() {
 	}
 
 	protected void setName(String s) {
-		name = (s.length() > 0 ? s : NodeL10n.getBase().getString("Bookmark.noName"));
+		name = (!s.isEmpty() ? s : NodeL10n.getBase().getString("Bookmark.noName"));
```

Comment:
Another case of negation in the ternary condition… again, I’m not sure we _need_ to change this…

Reply:
Reverted.  Unnecessarily confusing.